compiler/src/org.graalvm.compiler.core/src/org/graalvm/compiler/core/phases/LowTier.java

```diff
@@ -81,10 +99,30 @@ public LowTier(OptionValues options) {
 
         appendPhase(new UseTrappingNullChecksPhase());
 
+        if (Options.Autovectorize.getValue(options)) {
+            final String listValue = Options.AVList.getValue(options).trim();
+            List<String> list = new ArrayList<>();
+            if (!listValue.isEmpty()) {
+                list = Arrays.asList(listValue.split(","));
+            }
+
+            appendPhase(new IsomorphicPackingPhase(
```

Comment:
IPP currently runs in the low tier in part to avoid having to make changes to other phases to support the vector stamp types & nodes. Interested to get your opinion on this approach.